USAGE.md

```diff
@@ -0,0 +1,215 @@
+Usage instructions for oqsprovider
+==================================
+
+This file documents information required to properly utilize `oqsprovider`
+after installation on a machine running `openssl` v3.
+
+Beware that `oqsprovider` will not work on machines where an OpenSSL
+version below "3.0.0" is (the default) installed.
+
+## Baseline assumption
+
+An `openssl` version >= 3.0.0 is available and set in the "PATH" environment
+variable such as that the command `openssl version` yields a result documenting
+this, e.g., as follows:
+
+```
+OpenSSL 3.2.0-dev  (Library: OpenSSL 3.2.0-dev )
+```
+
+## Activation
+
+Every OpenSSL provider needs to be activated for use. There are two main ways
+for this:
+
+### Explicit command line option
+
+#### -provider
+
+Most `openssl` commands permit passing the option `-provider`: The name after
+this command is that of the provider to be activated.
+
+As an example: `openssl list -signature-algorithms -provider oqsprovider`
+outputs all quantum safe signature algorithms made available for `openssl` use.
+
+#### -provider-path
+
+All `openssl` commands accepting `-provider` also permit passing `-provider-path`
+as a possibility to reference the location in the local filesystem where the
+provider binary is located. This is of particular use if the provider did not
+(yet) get installed in the system location, which typically is in `lib/ossl-modules`
+in the main `openssl` installation tree.
+
+### Configuration file
+
+As an alternative to passing command line parameters, providers can be activated
+for general use by adding instructions to the `openssl.cnf` file. In the case of
+`oqs-provider` add these lines to achieve this:
+
+```
+[provider_sect]
+default = default_sect
+oqsprovider = oqsprovider_sect
+[oqsprovider_sect]
+activate = 1
+```
+
+#### module option
+
+Next to the "activate" keyword, `openssl` also recognizes the "module" keyword
+which mirrors the functionality of `-provider-path` documented above: This way,
+a non-standard location for the `oqsprovider` shared library (.SO/.DYLIB/.DLL)
+can be registered for testing.
+
+If this configuration variable is not set, the global environment variable
+"OPENSSL_MODULES" must point to a directory where the `oqsprovider` binary
+is to be found. 
+
+If the `oqsprovider` binary cannot be found, it simply (and silently) will
+not be available for use.
+
+#### System wide installation
+
+The system-wide `openssl.cnf` file is typically located at (operating system dependent):
+- /etc/ssl/ (UNIX/Linux)
+- /opt/homebrew/etc/openssl@3/ (OSX Homebrew)
```

Comment:
/opt/homebrew is the path for brew on macOS on Apple Silicon. On Intel it's /usr/local. 

Also it's now called "macOS", not "OSX" :)

Reply:
Thanks. Suitably fixed in https://github.com/open-quantum-safe/oqs-provider/pull/204/commits/2b597c79e3e73ed61f43149d0f99288adefa694d.